data-upgrade-news/src/main/java/org/exoplatform/news/upgrade/targets/NewsArticlesStatusUpgrade.java

```diff
@@ -0,0 +1,121 @@
+import org.exoplatform.social.metadata.model.Metadata;
+import org.exoplatform.social.metadata.model.MetadataType;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Query;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class NewsArticlesStatusUpgrade extends UpgradeProductPlugin {
+  private static final Log         LOG           = ExoLogger.getLogger(NewsArticlesStatusUpgrade.class.getName());
+
+  public static final MetadataType METADATA_TYPE = new MetadataType(4, "newsTarget");
+
+  private EntityManagerService     entityManagerService;
+
+  private NewsService              newsService;
+
+  private MetadataService          metadataService;
+
+  private PortalContainer          container;
+
+  private int                      insertedValue = 0;
+
+  public NewsArticlesStatusUpgrade(InitParams initParams,
+                                   EntityManagerService entityManagerService,
+                                   NewsService newsService,
+                                   MetadataService metadataService,
+                                   PortalContainer container) {
+    super(initParams);
+    this.entityManagerService = entityManagerService;
+    this.newsService = newsService;
+    this.metadataService = metadataService;
+    this.container = container;
+  }
+
+  public int getInsertedValue() {
+    return insertedValue;
+  }
+
+  @Override
+  public void processUpgrade(String oldVersion, String newVersion) {
+    long startupTime = System.currentTimeMillis();
+    LOG.info("Start upgrade of property item status of published news");
+
+    boolean transactionStarted = false;
+    PortalContainer container = PortalContainer.getInstance();
+    RequestLifeCycle.begin(container);
+    EntityManager entityManager = this.entityManagerService.getEntityManager();
+    try {
+      if (!entityManager.getTransaction().isActive()) {
+        entityManager.getTransaction().begin();
+        transactionStarted = true;
+      }
+      List<Metadata> metadataList = metadataService.getMetadatas(METADATA_TYPE.getName(), 0);
+
+      List<MetadataItemEntity> items = new ArrayList<>();
+      if (!metadataList.isEmpty()) {
+        for (Metadata metadataItem : metadataList) {
+          String sqlString = "SELECT * FROM SOC_METADATA_ITEMS WHERE METADATA_ID = '" + metadataItem.getId() + "'";
+          Query nativeQuery = entityManager.createNativeQuery(sqlString, MetadataItemEntity.class);
+          List<MetadataItemEntity> metadataItems = nativeQuery.getResultList();
+          metadataItems.forEach(item -> items.add(item));
+        }
+      }
+      News news = null;
+
+      if (items.isEmpty()) {
+        LOG.info("Metadata Items properties is empty");
+      } else {
+        List<MetadataItemEntity> metadataItemsList = items.stream().distinct().collect(Collectors.toList());
+        String sqlString2 = "DELETE FROM SOC_METADATA_ITEMS_PROPERTIES";
```

Comment:
- only metadata items with staged property and related to news targets should be deleted
- use a significant name